apps/project/schema.py

```diff
@@ -425,6 +426,13 @@ class Meta:
         filterset_class = ProjectByRegionGqlFilterSet
 
 
+class PublicProjectByRegionListType(CustomDjangoListObjectType):
+    class Meta:
+        model = Region
+        base_type = RegionWithProject
+        filterset_class = RegionFilterSet
```

Comment:
```suggestion
        filterset_class = ProjectByRegionGqlFilterSet
```

We also need to filter out private projects in projects_ids. This will need another `qs` method in the filterset.